src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesSpec.scala

```diff
@@ -51,23 +51,32 @@ class LeoRoutesSpec extends FlatSpec with ScalatestRouteTest with CommonTestData
 
   it should "200 when creating and getting cluster" in isolatedDbTest {
     val newCluster = ClusterRequest(Map.empty, Some(jupyterExtensionUri), Some(jupyterUserScriptUri), None, None, Some(UserJupyterExtensionConfig(Map("abc" ->"def"))))
+    val clusterNames = Seq(clusterName.value, s"${clusterName.value}-v2")
+    val baseUrl = "/api/cluster"
+    val versionedBaseUrls = Seq(baseUrl, s"$baseUrl/v2")
 
-    Put(s"/api/cluster/${googleProject.value}/${clusterName.value}", newCluster.toJson) ~>
-      timedLeoRoutes.route ~> check {
-      status shouldEqual StatusCodes.OK
-
-      validateCookie { header[`Set-Cookie`] }
+    // PUT endpoints have two versions
```

Comment:
Should probably add some cases to `LeonardoServiceSpec` as well to validate the new methods (unless that'll happen in the follow-on PR).

Reply:
Yea, I think it'll be more meaningful to add the cases to `LeonardoServiceSpec` as part of making sure the asynchronous operations do what they are supposed to do so that the `GET` endpoint returns the same results as for clusters created via `PUT v1`.